tests/rptest/tests/nodes_decommissioning_test.py

```diff
@@ -15,16 +15,19 @@
 from rptest.clients.types import TopicSpec
 from rptest.tests.end_to_end import EndToEndTest
 from rptest.services.admin import Admin
-from rptest.services.redpanda import CHAOS_LOG_ALLOW_LIST
+from rptest.services.redpanda import CHAOS_LOG_ALLOW_LIST, RESTART_LOG_ALLOW_LIST
 
 
 class NodesDecommissioningTest(EndToEndTest):
     """
     Basic nodes decommissioning test.
     """
     @cluster(num_nodes=6)
-    def test_decommissioning_working_node(self):
-
+    def test_decommissioning_working_node(
+        self,
+        # A decom can look like a restart in terms of logs from peers dropping
+        # connections with it
+        log_allow_list=RESTART_LOG_ALLOW_LIST):
```

Comment:
Maybe I'm missing something but where is log_allow_list used?

Reply:
Good catch, this was meant to be an argument to @cluster, fixed.